addOns/pscanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -108,7 +108,7 @@ <H2>Permissions Policy Header Not Set</H2>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/PermissionsPolicyScanRule.java">PermissionsPolicyScanRule.java</a>
 
 <H2>Source Code Disclosure</H2>
-Application Source Code was disclosed by the web server.
```

Comment:
This should be kept.

Reply:
@thc202 Added the notes back to help in latest commit. 
Thank you for guiding me and answering my questions! It seems I am doing some mess up with git (which is little bit new for me).

In latest commit/push, somehow git didn't allow me to use "fixup" when I rebase my local before push. I had to leave the previous commits as "pick". Hope this won't be an issue. If not, please advise how can I solve this issue. 

Once I understand this whole flow, I can work little bit faster.  Again, thank you very much for your help!

Reply:
After fetching from upstream you'll have to rebase on `upstream/main` now, the default branch was renamed recently. (You'll also have to resolve the conflicts.)

Reply:
@thc202 did these changes. Hope latest code is in acceptable/merge ready state. Let me know if anything missing or need to be modified. Thank you for your help and guidance!

Reply:
The space added at the beginning should be removed (for both sentences).